tests/foreman/cli/test_host.py

```diff
@@ -1,5 +1,8 @@
 """CLI tests for ``hammer host``."""
```

Comment:
nope, i'm using pylint too :) all the names here are ok, no need to silence pylint (at least for now)